packages/sanity/src/core/releases/components/__tests__/ReleaseActionsResolver.test.tsx

```diff
@@ -65,7 +67,7 @@ describe('ReleaseActionsResolver', () => {
           documents: mockDocuments,
         },
       }),
-      expect.any(Object),
+      undefined,
```

Comment:
Looking into this and it's kind of a hard one to understand why it's demanding the undefined. What I was able to find is the following: Apparently the second argument in React 19 calls the mock function with two arguments but the second is undefined instead of the react context object. 

I'm still trying to understand why beyond it being some cryptic React 19 thing, I will keep digging into it, but @jordanl17 since you are the person that originally wrote this tests does this ring any bells? Any thoughts? 
I looked into the ReleaseActions and it looks like they all work as expcted in this branch,